setup.py

```diff
@@ -50,27 +50,27 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-bigquery >= 0.20.0',
-    'google-cloud-bigtable >= 0.20.0',
-    'google-cloud-happybase >= 0.20.0',
-    'google-cloud-core >= 0.20.0',
-    'google-cloud-datastore >= 0.20.0',
-    'google-cloud-dns >= 0.20.0',
-    'google-cloud-error-reporting >= 0.20.0',
-    'google-cloud-language >= 0.20.0',
-    'google-cloud-logging >= 0.20.0',
-    'google-cloud-monitoring >= 0.20.0',
-    'google-cloud-pubsub >= 0.20.0',
-    'google-cloud-resource-manager >= 0.20.0',
-    'google-cloud-storage >= 0.20.0',
-    'google-cloud-translate >= 0.20.0',
-    'google-cloud-vision >= 0.20.0',
-    'google-cloud-runtimeconfig >= 0.20.0',
+    'google-cloud-bigquery >= 0.21.0',
```

Comment:
Are we doing this because we haven't figured out exactly how our release process should work?

These should all be at different versions right?